MobileWallet/Screens/Home/Transaction/TransactionViewControllerExtension.swift

```diff
@@ -94,19 +95,19 @@ extension TransactionViewController {
         valueContainerView.addSubview(valueImage)
 
         valueImage.centerYAnchor.constraint(equalTo: valueLabel.centerYAnchor).isActive = true
-        valueImage.trailingAnchor.constraint(equalTo: valueLabel.leadingAnchor, constant: -SIDE_PADDING / 2).isActive = true
+        valueImage.trailingAnchor.constraint(equalTo: valueLabel.leadingAnchor, constant: -Theme.shared.sizes.appSidePadding / 2).isActive = true
         valueImage.heightAnchor.constraint(equalToConstant: 21).isActive = true
         valueImage.widthAnchor.constraint(equalToConstant: 21).isActive = true
         valueImage.contentMode = .scaleAspectFit
     }
 
     func setFeeLabel(_ feeText: String) {
         valueContainerViewHeightConstraintFull.isActive = false
-        valueContainerViewHeightConstraintFull = valueContainerView.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: VALUE_VIEW_HEIGHT_MULTIPLIER_FULL, constant: 30)
+        valueContainerViewHeightConstraintFull = valueContainerView.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: valueViewHeightMultiplierFull, constant: 30)
```

Comment:
These can look better still with shorter lines, but it will cost new lines, and this is all layout code I guess, not sure if it has to be that readable.